presto-jdbc/src/main/java/io/prestosql/jdbc/PrestoDriverUri.java

```diff
@@ -135,6 +138,11 @@ public String getUser()
         return APPLICATION_NAME_PREFIX.getValue(properties);
     }
 
+    public String getClientTags()
```

Comment:
Move this method to be directly below `getExtraCredentials` so that the ordering is consistent everywhere